### ISSUE TITLE
Kuberhealthy status page is missing `CurrentMaster` field.

### ISSUE THREAD

**Issue by jonnydawg** [Comcast/kuberhealthy#648]
**Describe the bug**
The Kuberhealthy JSON status page reports a `blank` `CurrentMaster` field.

**Steps To Reproduce**
1. Install Kuberhealthy with some checks.
2. Let the checks run and complete.
3. Observe that the status page reports a blank `CurrentMaster` field.

**Expected behavior**
CurrentMaster should be reported as the name of the current Kuberhealthy master pod, but does not.

**Screenshots**
Soon™

**Versions**
 - Cluster OS: [e.g. Debian GNU/Linux 9 (stretch)]
 - Kubernetes Version: 1.11.7
 - Kuberhealthy v2.3.0

**Comment by integrii** [comment]
This may be connected to a bug we are seeing in the v2.3 image where we get a false check error that says something like `check deleted expectedly`.

**Comment by joshulyne** [comment]
> This may be connected to a bug we are seeing in the v2.3 image where we get a false check error that says something like `check deleted expectedly`.
error message is: `pod deleted expectedly`
the non-master kuberhealthy pod has the following logs: 
```
time="2020-09-16T03:03:22Z" level=debug msg="khstate custom resource found: resource-quota"
time="2020-09-16T03:03:22Z" level=debug msg="Retrieving khstate custom resource for: resource-quota"
time="2020-09-16T03:03:22Z" level=debug msg="Successfully retrieved khstate resource: resource-quota"
time="2020-09-16T03:03:22Z" level=debug msg="Setting execution state of check resource-quota to false [Check execution error: pod deleted expectedly] bd1ba474-7557-43d8-90c8-cc4f5136b703"
time="2020-09-16T03:03:22Z" level=debug msg="Checking existence of custom resource: resource-quota"
time="2020-09-16T03:03:22Z" level=debug msg="khstate custom resource found: resource-quota"
time="2020-09-16T03:03:22Z" level=debug msg="kuberhealthy resource-quota writing khstate with ok: false and errors: [Check execution error: pod deleted expectedly] at last run: 2020-09-16 03:03:22.890482206 +0000 UTC m=+3144510.042178847"
time="2020-09-16T03:03:22Z" level=debug msg="Successfully updated CRD for check: resource-quota in namespace kuberhealthy"
```
where you can see it updates the khstate with this `pod deleted expectedly` error for some reason. The non-master kuberhealthy pod should not be allowed to do that... Seems like there's some confusion there

**Comment by integrii** [comment]
This issue will be fixed when we merge PR #584
